spec/lib/open_project/custom_field_format_spec.rb

```diff
@@ -93,6 +101,15 @@
           .to contain_exactly("bool", "date", "float", "int", "link", "list", "string", "text", "user",
                               "version", "hierarchy", "empty")
       end
+
+      context "with calculated values feature flag enabled", with_flag: { calculated_value_project_attribute: true } do
```

Comment:
Nesting inside `custom_field_hierarchies` is not needed - enough to just have one test when nothing is enabled, one when hierarchies and one when calculated values
And it would probably be better to structure tests in `describe ".available_formats"` same as in `describe ".available_for_class_name"` - same idea, less verbose